pom.xml

```diff
@@ -5,4 +5,31 @@
   <version>0.0.1-SNAPSHOT</version>
   <name>car-app</name>
   <description>car-app</description>
+  
+  <dependencies>
+  
+  		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-entitymanager</artifactId>
```

Comment:
This dependency is deprecated, from version 5.2.x onward you can use only a single dependency "hibernate-core".
Put the version as a maven property: https://maven.apache.org/pom.html#Properties